tools/dump_tool/HowToUseDumpTool.md

```diff
@@ -265,13 +265,132 @@ Ramdump received successfully..!
 1. Run Ramdump Parser Script
 ```
 cd $TIZENRT_BASEDIR/tools/dump_tool/
 python dumpParser.py -r $TIZENRT_BASEDIR/build/output/bin/ramdump_0x02020000_0x0210c800.bin -e $TIZENRT_BASEDIR/build/output/bin/tinyara -g 0
 ```
 2. See the Output
 
+#### To display Debug Symbols/Crash point using assert logs
+1. Copy crash logs
+    First copy the crash logs to a file named log_file in tools/dump_tool/`<log_file>`
+2. Run Script in tools/dump_tool path
+    $ python3 ramdumpParser.py -e `<Tinyara elf path>` -t `<Log file path>`
+
+    ex)
+    $ python3 ramdumpParser.py -e ../../build/output/bin/app_dbg -t ./log_file
+
+Example Call Stack Output for App crash is as follows:
+```
+*************************************************************
+dump_file         : None
+log_file          : .log_file
+Tinyara_elf_file  : ../../build/output/bin/tinyara
+Tinyara_elf_file  : ../../../output/wifi_dbg
+*************************************************************
+
+Number of applicaions : 2
+App1 is : micom
+App2 is : wifi
+
+----------------------------------------------------------
+App Crash point is as follows:
+App name : wifi
+0x00001492
+main
+/root/tizenrt/loadable_apps/loadable_sample/wifiapp/wifiapp.c:85
+
+0x00003367
+binary_manager_send_request
+/root/tizenrt/framework/src/binary_manager/binary_manager_interface.c:97
+
+----------------------------------------------------------
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~  DEBUG DISPLAY SYMBOLS ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Dump_address	 Symbol_address	  Symbol_name	File_name
+0xe0072f3	 0xe007290 	  up_assert	/root/tizenrt/os/include/assert.h:211
+0xe04f0bn	 0xe04f028 	  __FUNCTION__.6571
+0xe04f043	 0xe04f028 	  __FUNCTION__.6571
+0xe007733	 0xe0076e8 	  up_memfault	/root/tizenrt/os/arch/arm/src/armv8-m/up_memfault.c:116
+0x10012345	 0x1001232c 	  up_doirq	/root/tizenrt/os/arch/arm/src/armv8-m/up_doirq.c:91
+0x10012055	 0x10012014 	  exception_common	/root/tizenrt/os/arch/arm/src/armv8-m/up_exception.S:156
+0xe001e4f	 0xe001e4e 	  sem_freeholder	/root/tizenrt/os/kernel/semaphore/sem_holder.c:163
+
+PC_value	 Symbol_address	  Symbol_name	File_name
+0x218ec12	 0x2000000 	  __psram_image2_text_start__
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+```
+Example Call Stack Output for App crash is as follows:
+```
+*************************************************************
+dump_file         : None
+log_file          : log_file
+Tinyara_elf_file  : ../../../output/app_dbg
+*************************************************************
+
+Number of applicaions : 1
+App[1] is : app
+
+----------------------------------------------------------
+App Crash point is as follows:
+App name : app
+0x0000001a
+main
+/root/product/.tizenrt/loadable_apps/product_apps/default_app/app.c:45
+
+----------------------------------------------------------
+
+--------------------------- DEBUG SYMBOLS IN KERNEL TEXT RANGE -------------------------
```

Comment:
As my understanding, the crash happens from app.
But in this log, it says `DEBUG SYMBOLS IN KERNEL TEXT RANGE`.
Please check this.

Reply:
Please change the md file also based on your changes.

Reply:
Updated both commit details and the md file